changes/2024-TBD_key-store-admin/proposed.smithy

```diff
@@ -0,0 +1,172 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
Where is this going?
Is it a new MPL subproject? i.e. `<MPL root>/KeyStoreAdmin`
Or is it a new localService inside the MPL-Core subproject? i.e. `<MPL root>/AwsCryptographicMaterialProviders/KeyStoreAdmin`

Reply:
Discussed in meeting.
New Local Service of the MPL, alongside the Key Store.